src/api.c

```diff
@@ -1752,6 +1774,33 @@ char *cg_build_path_locked(const char *name, char *path, const char *type)
 			if (name) {
 				char *tmp;
 
+				/*
+				 * If the user has passed absolute cgroup path,
+				 * avoid using systemd delegate slice as path
+				 * prefix.
+				 */
+				if (name[0] == '/') {
```

Comment:
This feels kinda klunky.  So we build a path above, but if the first char is `/`, we'll override that path and put a new one in instead?